system-tests/jobs/test-jobs.js

```diff
@@ -7,7 +7,7 @@ describe("Jobs", function() {
     });
   });
 
-  it("Create a simple job", function() {
+  it("should create a simple job", function() {
```

Comment:
is it really necessary? I believe the wording `should` is not a worth "pattern". Not a big deal though.